Remotion/Web/Development.WebTesting/WebDriver/Configuration/Chromium/Edge/EdgeBinariesProvider.cs

```diff
@@ -0,0 +1,171 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.IO.Compression;
+using System.Net;
+using JetBrains.Annotations;
+
+namespace Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chromium.Edge
+{
+  /// <summary>
+  /// Provides Edge and msedgedriver binaries and a custom user directory.
+  /// </summary>
+  public class EdgeBinariesProvider
+  {
+    private const string c_webDriverFolderName = @"Remotion.Web.Development.WebTesting.WebDriver\msedgedriver";
+
+    private const string c_driverDownloadUrlFormat = "https://msedgedriver.azureedge.net/{0}/edgedriver_win32.zip";
+
+    private const string c_driverExecutableName = "msedgedriver.exe";
+    private const string c_edgeExecutableName = "msedge.exe";
+    private const string c_zipFileName = "msedgedriver.zip";
+
+    private static readonly Version s_minimumSupportedEdgeVersion = new Version (76, 0);
+
+    /// <summary>
+    /// Returns the <see cref="EdgeExecutable"/> that contains the installed Edge browser location, the corresponding msedgedriver location,
+    /// and a temporary user directory. If no msegedriver exists, it is downloaded.
+    /// </summary>
+    public EdgeExecutable GetInstalledExecutable ()
+    {
+      var browserPath = GetInstalledEdgePath();
+      var driverPath = GetDriverPathAndDownloadIfMissing (browserPath);
+      var userDirectory = GetUserDirectoryTempPath();
+
+      return new EdgeExecutable (browserPath, driverPath, userDirectory);
+    }
+
+    private string GetInstalledEdgePath ()
+    {
+      var defaultBetaEdgePath = Path.Combine (Get32BitProgramFilesPath(), "Microsoft", "Edge Beta", "Application", c_edgeExecutableName);
```

Comment:
Is there a follow up Jira Ticket for removing Edge Beta? Or should this be kept in the Code?